manifests/y/yuyuanweb/jianqiebanzhushou/0.6/yuyuanweb.jianqiebanzhushou.installer.yaml

```
@@ -0,0 +1,24 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: yuyuanweb.jianqiebanzhushou
+PackageVersion: "0.6"

```

```suggestion
PackageVersion: 0.0.6
```
Please update file path to match this change.